### DISCUSSION TITLE
Region Missing

### DISCUSSION THREAD

**Discussion by ManFromNapa** [code4recovery/12-step-meeting-list#959] [Q&A] [answered]
I've got everything running, but for some reason the "Region" isn' populating for most of my meetings.  I've looked my sheet over to find some sort of typo, but can' find any pattern to why only a few meetings are importing regions.

**Comment by joshreisner** [comment, marked as answer]
i think it’s likely there might be some inconsistencies in the sheet. my suggestion would be to sort it by address and confirm that the location and region columns are the same for each meeting at a given address

if that doesn’t work please upload it to a google sheet and we can take a look together

**Reply by ManFromNapa** [reply]
I'm looking for inconsistencies, or typos, or even things like spaces before the region, but I'm no finding them.  Out of 89 meetings, only around 6 have the region data after import.  Here's a link to my Google Sheet:

https://docs.google.com/spreadsheets/d/1l6VM8FSxcR1BxCKLgtFJHGs3pVkpJ9A97MRE-t-2aVk/edit?usp=sharing

**Reply by joshreisner** [reply]
great. try changing regions in the heading to region

"regions" only currently works with TSML UI. importing to TSML via CSV, you need to use "region" and "sub region"

**Reply by ManFromNapa** [reply]
Made the change to my google sheets.  Still only a few meetings have a region after import.

I'm using the Google Sheets Importer to convert to JSON.

<img width="1146" alt="image" src="https://user-images.githubusercontent.com/93745225/197919625-0e63f741-cbe8-4493-926d-4a1e089e5517.png">

**Reply by joshreisner** [reply]
ah, ok that won't work. the sheets importer is designed to work with TSML UI, not TSML. why did you want to import it to WordPress if Google Sheets is your source of truth?

**Reply by ManFromNapa** [reply]
Because from everything I read that's what it looked like I should do. When looking for the documentation and inspecting the example google doc there seemed to be a requirement for the JSON backend. 

Do I just create a "Meetings" page and add the sample code in it?

**Reply by joshreisner** [reply]
Ah yes, sorry it's not clear. If you are using a Google Sheet as your database, you don't need TSML at all, I would go ahead and uninstall the plugin from WordPress.

And yes, I would then create a meetings page and drop in the HTML code.

**Reply by ManFromNapa** [reply]
I got it.  Thanks.
